db/migrate/20220729161012_change_default_of_enabled_of_products.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeDefaultOfEnabledOfProducts < ActiveRecord::Migration[7.0]
+  def change
+    change_column :products, :enabled, :boolean, default: false
```

Comment:
What will happen if I try to reverse this migration? Will that work?
Can we use something isntead of change_column?

Reply:
Use change_default